phoenix-core/src/it/java/org/apache/phoenix/end2end/json/JsonFunctionsIT.java

```diff
@@ -633,4 +637,34 @@ public void testJsonQuery() throws Exception {
             compareJson(rs.getString(5), JsonDatatypes, "$.datatypes.mixedArray");
         }
     }
+
+    @Test
+    public void testExpressionIndex() throws Exception {
+        Properties props = PropertiesUtil.deepCopy(TEST_PROPERTIES);
+        String tableName = generateUniqueName();
+        String indexName = "IDX_" + generateUniqueName();
+        try (Connection conn = DriverManager.getConnection(getUrl(), props)) {
+            conn.setAutoCommit(true);
+            String
+                    ddl =
+                    "create table if not exists " + tableName +
+                            " (pk integer primary key, col integer, jsoncol.jsoncol json)";
+            conn.createStatement().execute(ddl);
+            conn.createStatement().execute(
+                    "UPSERT INTO " + tableName + " (pk, col, jsoncol) VALUES (1,2, '" + JsonDoc1 + "')");
+            conn.createStatement().execute(
+                    "CREATE INDEX " + indexName + " ON " + tableName
+                            + " (JSON_VALUE(JSONCOL,'$.type'), JSON_VALUE(JSONCOL,'$.info.address.town')) include (col)");
```

Comment:
Is it possible to have a JSON function in included columns for an index ? Can we add a test for the same

Reply:
I remember discussing this with @kadirozde . I don't think we support functions in include columns currently. Instead we can use uncovered index to achieve what we want with functions in include columns.

Reply:
Tested it this way:
`0: jdbc:phoenix:localhost> create index indexmytable1 on mytable1 (colA) include (PHOENIX_ROW_TIMESTAMP());
Error: ERROR 601 (42P00): Syntax error. Encountered "(" at line 1, column 77. (state=42P00,code=601)`

`0: jdbc:phoenix:localhost> create index idx_jsontest6 on jsontest6 (JSON_VALUE(JSONCOL,'$.type'), JSON_VALUE(JSONCOL,'$.info.address.town')) include (JSON_VALUE(JSONCOL, '$.info.address.county'));

Error: ERROR 601 (42P00): Syntax error. Encountered "(" at line 1, column 134. (state=42P00,code=601)`